test/common/router/config_impl_test.cc

```diff
@@ -2171,17 +2171,17 @@ TEST(RouteMatcherTest, DirectResponse) {
       "domains": ["redirect.lyft.com"],
       "routes": [
         {
-          "prefix": "/foo",
```

Comment:
Sorry, I'm probably missing something obvious, but why change these?
It's generally helpful when changing code to leave the existing tests as is to make it clear there's no change to existing functionality.

Reply:
It seemed to me to be better to have the path prefixes mean something, especially as more kinds of redirects were added.

So there would be...
```
/host => host_redirect
/path => path_redirect
/host_path => host_redirect + path_redirect
...
```
...instead of `foo`/`bar`/`baz` which don't mean anything. That was slightly complicated by `/host` being a prefix of `/host_path` so ordering matters now 😕.

I can try rework or go back to a minimal change?

Reply:
Fair enough - I'll take a close look at diffs when I do my pass, so please just consider it for future PRs.

Reply:
Thanks 🙏🏼

Reply:
Thanks for the explanation - it is good clean up :-)